lib/engines/content_block_manager/app/components/content_block_manager/content_block/document/show/document_timeline/timeline_item_component.rb

```diff
@@ -13,16 +13,27 @@ def initialize(version:, schema:, is_first_published_version:, is_latest:)
   attr_reader :version, :schema, :is_first_published_version, :is_latest
 
   def title
-    case version.state
-    when "published"
+    if version.is_embedded_update?
+      "#{updated_subschema.name.singularize} created"
+    elsif version.state == "published"
       is_first_published_version ? "#{version.item.block_type.humanize} created" : version.state.capitalize
-    when "scheduled"
+    elsif version.state == "scheduled"
       "Scheduled for publishing on #{version.item.scheduled_publication.to_fs(:long_ordinal_with_at)}"
     else
       "#{version.item.block_type.humanize} #{version.state}"
     end
   end
 
+  def updated_subschema
+    @updated_subschema ||= @schema.subschema(version.event_details["updated_embedded_object_type"])
```

Comment:
do we go and get the schema here in case the embedded object name has changed? Wondering if would be better to use the name at the time the version is created, because otherwise schemas will always have to be backwards compatible or versioned?